### ISSUE TITLE
Layout: wcag issues

### ISSUE THREAD

**Issue by kristiina-richards** [TEDI-Design-System/react#344]
### Description of the problem

1. When using it on 1280px width then 400% zoom has a horizontal scroll. horizontal scroll is fine with tables but not on the whole page at that zoom - caused by long email address in footer
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/reflow.html)
2. JAWS doesn't read menu items as clickable. role='navigation' should be more appropriate
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)
